custom/icds_reports/sqldata/base.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import, unicode_literals
+
+import uuid
+
+from corehq.apps.reports.sqlreport import DatabaseColumn, SqlData
+from corehq.toggles import ICDS_COMPARE_QUERIES_AGAINST_CITUS, NAMESPACE_OTHER
+
+
+class IcdsSqlData(SqlData):
+    engine_id = 'icds-ucr'
+
+    def get_data(self, start=None, limit=None):
+        from custom.icds_reports.tasks import run_citus_experiment_raw_sql
+
+        for query in self.get_sql_queries():
```

Comment:
I don't think these include parameters, you should use `self.filter_values` as the params

Reply:
This was a bit tricker than that as it was not returning  postgresql paramertized queries like django was returning. (It returned something like `select * from table where property = :property` instead of `%(property)s`.) To keep the same interface I had to dig a bit into sqlagg and sqlalchemy.  https://github.com/dimagi/commcare-hq/pull/24098/commits/b26c7b50a6c5d6b075f8edfd7da82d3fd583e3ed#diff-515ee24d8bc1a09c528fcbf4764d4757R22

From my perspective it should be short lived enough that it shouldn't need to be publicly accessible in sqlagg (and need a new sql-agg release) but if we think it will be useful I can push some of this logic into the library